### ISSUE TITLE
not yet supported outside strict mode (nodejs 5.9.1)

### ISSUE THREAD

**Issue by kepeto** [yagop/node-telegram-bot-api#311]
> Question

hi,

for some reason i cannot upgrade my nodejs from 5.9.1 to latest version (using windows server2003)

how to resolve this error?

(trying to use 'use strict' but it doesn't help)

```
D:\projects\node_modules\node-telegram-bot-api\src\telegram.js:35
class TelegramBot extends EventEmitter {
^^^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet sup
ported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
```

thanks

**Comment by GochoMugo** [comment]
I believe we have already fixed this issue. We just forgot to publish the fixes. Sorry.

See issue #297.

**Comment by kepeto** [comment]
@GochoMugo we will wait the fix to be published
thanks!

**Comment by GochoMugo** [comment]
Fixed in **v0.12.1**.
